packages/client/game/graphics.d.ts

```diff
@@ -1445,7 +1445,7 @@ declare interface GameGraphics extends GameGraphicsLegacy {
    * 	mp.game.graphics.setEntityOverlayPassEnabled(true);
 	 *
 	 *	let batch = mp.game.graphics.createEntityOverlayBatch(overlayParams);
- 	 *
+	   *
```

Comment:
```suggestion
   *
```
This one looks to be mistaken changed